homeassistant/components/tradfri/const.py

```diff
@@ -1,7 +1,25 @@
 """Consts used by Tradfri."""
+from homeassistant.components.light import SUPPORT_TRANSITION, SUPPORT_BRIGHTNESS
 from homeassistant.const import CONF_HOST  # noqa pylint: disable=unused-import
 
 CONF_IMPORT_GROUPS = "import_groups"
```

Comment:
Do we order this list alphabetically?

Reply:
It's nice to do that, but it's not something we enforce.